packages/package-store/src/storeController/prune.ts

```diff
@@ -17,6 +18,9 @@ export default async function prune (storeDir: string) {
   const dirs = await fs.readdir(cafsDir)
   let fileCounter = 0
   for (const dir of dirs) {
+    if (skipDirs.includes(dir)) {
+      continue
+    }
     const subdir = path.join(cafsDir, dir)
```

Comment:
the actual issue is that `fs.readdir` lists all the items in the directory (not only directories but files also). So the proper solution would be to check if subdir is actually a directory before running `fs.readdir(subdir)`.

Seems like we may use `fs.readdir(cafsDir, { withFileTypes: true })` at line 18 and then filter out the items that are `isDirectory() === false`